rows/plugins/sqlite.py

```diff
@@ -64,6 +64,23 @@ def _get_connection(filename_or_connection):
         return filename_or_connection
 
 
+def _get_default_table_name(filename_or_connection):
+    connection = _get_connection(filename_or_connection)
+    select_tables = "SELECT name FROM sqlite_master WHERE type='table';"
+    current_tables = (x[0] for x in connection.execute(select_tables).fetchall())
+    table_name_raw = 'rows_{}'
+
+    i = 1
+
+    while True:
+        table_name = table_name_raw.format(i)
+        if not table_name in current_tables
```

Comment:
`SyntaxError` here (missing comma).
You can check these errors by running the tests with:

    make test